University-advisor/Forms/MainForm.cs

```diff
@@ -102,17 +101,11 @@ private void ChangeStatus_Click(object sender, EventArgs e)
 
         private void SetValues()
         {
-            var universityResult = SqlDriver.Fetch("SELECT name FROM universities");
-            if (universityResult.Count != 0)
-            {
-                var universityList = new List<string>();
-                foreach (Dictionary<string, object> row in universityResult)
-                {
-                    universityList.Add(row["name"].ToString());
-                }
-                universityBox.DataSource = universityList;
-                statusBox.DataSource = statusList;
-            }
+            UserEditingService service = new UserEditingService(currentUser);
+            universityBox.DataSource = service.GetAllUniversities();
+            universityBox.SelectedItem = service.GetCurrentUniversity();
+            statusBox.DataSource = statusList;
```

Comment:
I guess statusList could be moved to UserEditService and could be called in a similar way to earlier lines ex.: service.GetStatusList().